packages/scripts/src/create-webpack-configs.ts

```diff
@@ -276,8 +276,10 @@ export function createWebpackConfigForExteranlFeature({
             path: outputPath,
             filename: `[name].${target}.js`,
             chunkFilename: `[name].${target}.js`,
-            libraryTarget: 'var',
-            jsonpFunction: packageName + name,
+            library: {
```

Comment:
this change is probably why the tests are failing. need to figure out how this has to be configured on webpack@5.